phantasm-generator/pom.xml

```diff
@@ -40,4 +40,15 @@
             <artifactId>junit</artifactId>
         </dependency>
     </dependencies>
+    <build>
```

Comment:
phantasms are generated outside of database.active, so we need to know this at least doesn't blow up.